travis-deploy.sh

```diff
@@ -12,8 +12,15 @@ function goreleaser() {
     curl -sL https://git.io/goreleaser | bash
 }
 
-if [ ! -z "$TRAVIS_TAG" ]; then
+function gcr_push() {
+    openssl aes-256-cbc -K $encrypted_222a2009ef6d_key -iv $encrypted_222a2009ef6d_iv -in heptio-images-c14f11347d8b.json.enc -out heptio-images-c14f11347d8b.json -d
+    gcloud auth activate-service-account --key-file heptio-images-c14f11347d8b.json
+    # https://github.com/travis-ci/travis-ci/issues/9905
```

Comment:
This issue was closed. Do we still need to do the `unset` below?

Reply:
/shrug

I was just adding back the exact code that we previously had done this work. Let's just leave it (if it works) since this is all probably temporary anyways.

Reply:
> this is all probably temporary anyways
That is a very good point. Ignore that comment then :)